src/node_crypto.cc

```diff
@@ -4132,6 +4212,14 @@ void Sign::SignFinal(const FunctionCallbackInfo<Value>& args) {
   size_t buf_len = Buffer::Length(args[0]);
   char* buf = Buffer::Data(args[0]);
 
+  int padding = RSA_PKCS1_PADDING;
+  int saltlen = -2;
```

Comment:
Mh, sorry if I wasn’t clear, but my idea was that you could add `#include "node_constants.h"` at the top of this file and then use the constants here where you currently use the magic numbers directly